src/etc/pre-commit.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/env bash
+#
+# Call `tidy --bless` before each commit
+# Copy this scripts to .git/hooks to activate,
+# and remove it from .git/hooks to deactivate.
+#
+# For help running bash scripts on Windows,
+# see https://stackoverflow.com/a/6413405/6894799
```

Comment:
Hmm, maybe a good follow-up would be to have a version of this for powershell too. No need to add it now though.

Reply:
From what I can tell, git only understands unix-like scripts, and won't run e.g. `pre-commit.ps1`. I suspect that if the user has git installed, they'll be able to run git scripts as well, so actually this line is redundant...I think...I haven't used git on windows in a while.

Reply:
cc @mati865 - do you know if this will work on windows, and if not, how to make it work?

Reply:
@jyn514 Git for Windows ships bash shell.
I haven't tried using hooks there but they should work.

I hate powershell so personally I use bash/zsh everywhere, even on Windows.